content/docs/user-guide/how-to/update-tracked-files.md

```diff
@@ -1,76 +1,72 @@
-# Updating Tracked Files
+# How to Update Tracked Files
```

Comment:
Files or Data - in some case we use Data, in some cases Files. is it okay?

Reply:
Data is better but this is a pre-existing use case and renaming it was outside of the scope here esp. since it requires a redirect.

TBH I want to move other guides into this section: Merge Conflicts and Setup a Google Drive Remote for starters. So should we do all that in a separate PR?

Reply:
> So should we do all that in a separate PR? 

yes, def not this PR!

> Setup a Google Drive Remote

this should part of a bigger section that has a page per remote type. It's not How to

> Merge Conflicts

It can be probably part of the `How to` indeed.

Reply:
> Setup a Google Drive Remote...
should part of a bigger section that has a page per remote type. It's not How to

I think it's definitely "how-to" material, even if we put that remote config section somewhere else. I'll let it be for now then.

Reply:
You can put pretty much anything into How to this way ... How to manage external data, How to contribute, How to optimize deal with large data, How to run on Windows, etc ...


I think there should be something else that defines this section - closer to FAQ? some specific workflow questions?

In case of Google Drive - we'll have one page per remote. Even size in this case makes a bad fit. Eventually it might become bigger than everything else.

Reply:
Continued in #1976